test/nmodl/test_function_table.py

```diff
@@ -8,8 +11,14 @@
 def test_function_table():
     k3st_cell = k3stCell()
     k3st_cell.record()
-    k3st_cell.simulate(1, 0.1)
-    assert np.isclose([k3st_cell.record_vectors["tau1_rec"][2]], [0.097022])
+    coreneuron_enable = bool(
+        distutils.util.strtobool(os.environ.get("NRN_CORENEURON_ENABLE", "false"))
+    )
+    coreneuron_gpu = bool(
+        distutils.util.strtobool(os.environ.get("CORENRN_ENABLE_GPU", "false"))
+    )
+    k3st_cell.simulate(1, 0.1, coreneuron_enable, coreneuron_gpu)
+    assert np.isclose([k3st_cell.record_vectors["tau1_rec"][2]], [0.117614])
```

Comment:
Ι'm wondering whether this is because the initial voltage is different between the old and new way to run the simulations

Reply:
The issue was in the end calling `h.dt` in the wrong place. It should be called after `h.stdinit()`